Conformance -> understanding (#195)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

An exemplary syntax for statements of preference is defined in {{format}}.

## Understanding Preferences {#understanding}

This document and {{ATTACH}}
describe how statements of preference are associated with assets.

The goal of these specifications is to ensure
that the recipient of an asset knows
what preferences have been associated with the asset.
What a recipient then does with that information depends on many factors;
see {{applicability}}.

There are also some caveats that need to be considered
as it relates to understanding what preferences are
(as opposed to what actions might then follow).

> A recipient chooses the methods of expressing preferences it recognizes.

Author: ```suggestion
A recipient can only understand preferences expressed
through mechanisms it has implemented.
```

An exemplary syntax for statements of preference is defined in {{format}}.

## Understanding Preferences {#understanding}

This document and {{ATTACH}}
describe how statements of preference are associated with assets.

The goal of these specifications is to ensure
that the recipient of an asset knows
what preferences have been associated with the asset.
What a recipient then does with that information depends on many factors;
see {{applicability}}.

There are also some caveats that need to be considered

> as it relates to understanding what preferences are

Reviewer: ```suggestion
as it relates to understanding what the preferences for a given asset are
```

(as opposed to what actions might then follow).

A recipient can only understand preferences expressed
through mechanisms it has implemented.
Those methods might be limited to those in {{ATTACH}}
or it could also include other methods (see {{Section 1.3 of ATTACH}}).
If a preference is associated with an asset
using a method the recipient does not understand or recognize,
the recipient will remain ignorant of that preference.

Depending on the way in which preferences are expressed,

> a recipient might be unable to tell who has expressed a preference.

Reviewer: ```suggestion
a recipient might be unable to tell the source of the preference.
```

Reviewer: `who` implies a form of identity that I don't think we want to introduce

Unless the source is explicitly identified,
no assumptions can be made about where a preference originates.
For example, preferences in robots.txt (see {{Section 3 of ATTACH}})

> only implies that the operator of a server

Reviewer: ```suggestion
only implies that a server
```

is the source of those preferences.

A method of associating preferences with assets

> could explicitly define who is responsible for preferences,

Reviewer: ```suggestion
could explicitly define the source of the preferences,
```
